File: 2A_1/python/czy_parzysta.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  czy_parzysta.py
  

def main(args):
    liczba = int(input("Podaj liczbę: "))
    if liczba % 2 == 0:
        print("liczba parzysta")
    else:
        print("liczba nieparzysta")

    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: 2A_1/cpp/czy_parzysta.cpp
/*
 * czy_parzysta.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int liczba = 0;  // deklaracja i inicjacja zmiennej
	cout << "Podaj liczbę: ";
	cin >> liczba;

	if (liczba % 2 == 0) {
		cout << "liczba parzysta";
	} else {
		cout << "liczba nieparzysta";
	}
	
	return 0;
}

